app/utils/print/playback/sections/LeadTrusteeIndividual.scala

```diff
@@ -42,8 +42,9 @@ object LeadTrusteeIndividual {
             addressAnswers(index, userAnswers, countryOptions, name).flatten ++
             Seq(yesNoQuestion(TrusteePassportIDCardYesNoPage(index), userAnswers, "trusteePassportOrIdCardYesNo", name),
               passportOrIdCardQuestion(TrusteePassportIDCardPage(index), userAnswers, "trusteePassportOrIdCard", name, countryOptions),
```

Comment:
Passport details should be before address as these are an alternative to NINO

Reply:
Applied.